uiServices/src/testCaseServices.go

```diff
@@ -241,3 +241,30 @@ func getTestCase(rw http.ResponseWriter, req *http.Request) {
 
 }
 
+
+func deleteTestCase(rw http.ResponseWriter, req *http.Request){
+	testCasePathDir := getTestCaseHeader(req)
+	testCaseName := chi.URLParam(req, "testCaseName")
+	ValidateFileNameAndHeader(rw, req, testCasePathDir, testCaseName)
```

Comment:
You should handle the return from the `ValidateFileNameAndHeader ` as if validation fails you should return instead you are continuing with execution.
See how it is done in `deleteTestSuite` function.
You should also create a card to fix this everywhere as you have it in multiple places.

Reply:
You need to add a check here in our `deleteTestCase `